pom.xml

```diff
@@ -209,6 +209,7 @@ Jean-Yves Tinevez and Michael Zinsmaier.</license.copyrightOwners>
 		<dependency>
 			<groupId>net.imglib2</groupId>
 			<artifactId>imglib2</artifactId>
+			<version>4.6.1</version>
```

Comment:
Good catch @imagejan 
This was necessary at the time of the PR, but we don't need this anymore, now.

Reply:
If you need to override the version of a component in the future, please use a version property as @imagejan described, rather than hardcoding it in the `<version>` itself. Otherwise, the melting-pot script and other tooling will be unable to override it and some kinds of version skew will not be caught by the unified builds.

Reply:
Will do (actually, already doing that in most of my projects).